packages/website-backend/src/middleware/securityMiddleware.ts

```diff
@@ -89,5 +89,5 @@ export function passportAuthenticateGithub(
   res: express.Response,
   next: express.NextFunction
 ) {
-  return passport.authenticate('github')(req, res, next);
+  return passport.authenticate('github', { session: false })(req, res, next);
```

Comment:
This change doesn't apply only locally. What is the impact of this change? Will users have to login often because of this?

Reply:
I believe this does not change the working, but I'm not sure. Without this the user will not be able to log in at all as it gives an error in the back-end.

Reply:
I can login fine on acceptance and the e2e tests run fine, can you not?

Also here's the background for the breaking change that likely leads to this issue: https://medium.com/passportjs/fixing-session-fixation-b2b68619c51d 

We should probably implement the suggestions in the post, looks like logout might be broken right now however I thought I tested that before merging the passport change.

Reply:
Mhm behavior doesn't seem any different between acceptance and production so I agree that there likely isn't any impact. Perhaps we're not using sessions right now anyway. @nicojs what do you know about this?

Reply:
We're not using sessions, no.